ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,224 @@
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66A9', source: 'Phoinix', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P3N Ashen Eye',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66AB', source: 'Sparkfledged' }),
+      alertText: (data, matches, output) => {
+        if (!data.ashenEyeDirections)
+          data.ashenEyeDirections = [];
+        switch (matches.heading) {
+          case '-3.14':
+            data.ashenEyeDirections.push('north');
+            break;
+          case '-1.57':
+            data.ashenEyeDirections.push('west');
+            break;
+          case '0.00':
+            data.ashenEyeDirections.push('south');
+            break;
+          case '1.57':
+            data.ashenEyeDirections.push('east');
+            break;
+          default:
+            console.error('Ashen Eye: Got unforeseen ashen eye direction', matches.heading);
+            break;
+        }
+        if (data.ashenEyeDirections.length === 2) {
+          let safeSpot = '';
+          let first = '';
+          const dir1 = data.ashenEyeDirections[0];
+          const dir2 = data.ashenEyeDirections[1];
+          switch (dir1) {
+            case 'north':
+              safeSpot = output.s!();
+              break;
+            case 'west':
+              safeSpot = output.e!();
+              break;
+            case 'south':
+              safeSpot = output.n!();
+              break;
+            case 'east':
+              safeSpot = output.w!();
+              break;
+          }
+          switch (dir2) {
+            case 'north':
+              first = output.s!();
+              break;
+            case 'west':
+              first = output.e!();
+              break;
+            case 'south':
+              first = output.n!();
+              break;
+            case 'east':
+              first = output.w!();
+              break;
+          }
+          return first + ' then ' + safeSpot;
+        } else if (data.ashenEyeDirections.length > 3) {
+          data.ashenEyeDirections = [];
+        }
+      },
+      outputStrings: {
+        n: Outputs.north,
+        e: Outputs.east,
+        w: Outputs.west,
+        s: Outputs.south,
+      },
+    },
+    {
+      id: 'P3N Devouring Brand',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '669E', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Split Intercardinals',
+        },
+      },
+    },
+    {
+      id: 'P3N Searing Breeze',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B6', source: 'Phoinix', capture: false }),
+      response: Responses.stackThenSpread(),
+    },
+    {
+      id: 'P3N Spread Marker',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: ['008B'] }),
```

Comment:
```suggestion
      netRegex: NetRegexes.headMarker({ id: '008B' }),
```

Reply:
Changed in 1be9b4e868f8ab0e3c7278e054e1fb9fb7df5be7